client/Modal/types/PlaylistSelector/index.styl

```diff
@@ -1,24 +1,27 @@
-@import "~stylus/color.styl"
+@import "~stylus/settings/color.styl"
 
 .playlistSelector
-  height 500px
-  width 100%
-  .playlists
+    height 500px
+    width 100%
+
+.playlists
     display flex
     flex-direction column
     height 100%
     overflow-y scroll
     background-color @background-color
-    .playlist
-      cursor pointer
-      font-size calc(0.5vw + 10px)
-      padding 10px
-      margin-bottom 1px
-      background-color #4d4d4d
-      border 1px solid transparent
-      &:nth-child(odd)
+
+.playlist
```

Comment:
Is `playlist` not a child of `playlists`?

Reply:
No need to nest classes if React is renaming them to be unique